libraries/rush-sdk/webpack.config.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-env es6 */
```

Comment:
This can happen in a follow-up, but a lot of this should be able to be deduplicated with the webpack config in rush-lib.